src/dropdown/dropdown.component.ts

```diff
@@ -34,7 +34,9 @@ import { I18n } from "./../i18n/i18n.module";
 @Component({
 	selector: "ibm-dropdown",
 	template: `
-	<div class="bx--list-box">
+	<div
+		class="bx--list-box"
+		[ngClass]="{'bx--dropdown--light': light}">
```

Comment:
With the change this would become
```suggestion
		[ngClass]="{'bx--dropdown--light': theme === 'light'}">
```